### PR TITLE
Add docs to mention 1.27 support

### PR DESCRIPTION
**Pull request by ahreehong**
*Issue #, if available:*

*Description of changes:*
Update docs to mention kubernetes 1.27 support

*Testing (if applicable):*

### CODE REVIEW

File: docs/content/en/docs/reference/support/support-versions.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The following table shows EKS Anywhere version support for different Kubernetes
 
 | Kubernetes version | Supported EKS Anywhere version | First supported    | End of support   |
 |--------------------|---------------|--------------------|------------------|
+| 1.27               | 0.16 | June 1, 2023      |
```
Comment (ahreehong) [inline docs/content/en/docs/reference/support/support-versions.md:23]: How do we determine the `end of support` date?

**Review by taneyland — APPROVED** [review]

**Pull request merged**